### PR TITLE
ccmlib/node: add `env` argument to `Node.run_cqlsh()`

### PR DESCRIPTION
**Pull request by fruch**
for enabling more of dtest need, added `env` argument to 
pass in environment variables needed for the test.

### CODE REVIEW

File: ccmlib/node.py [hunk 1]
```diff
@@ -863,12 +863,15 @@ def run_cli(self, cmds=None, show_output=False, cli_options=[]):
                         print(log, end='')
                     i = i + 1
 
-    def run_cqlsh(self, cmds=None, show_output=False, cqlsh_options=None, return_output=False, timeout=600):
+    def run_cqlsh(self, cmds=None, show_output=False, cqlsh_options=None, return_output=False, timeout=600, env=None):
```
Comment (tchaikov) [inline ccmlib/node.py:866]:
```suggestion
    def run_cqlsh(self, cmds=None, show_output=False, cqlsh_options=None, return_output=False, timeout=600, extra_env=None):
```

File: ccmlib/node.py [hunk 2]
```diff
@@ -863,12 +863,15 @@ def run_cli(self, cmds=None, show_output=False, cli_options=[]):
...
         cqlsh_options = cqlsh_options or []
         cqlsh = self.get_tool('cqlsh')
         if not isinstance(cqlsh, list):
             cqlsh = [cqlsh]
-        env = self.get_env()
+        if env:
+            env = self.get_env().update(env)
+        else:
+            env = self.get_env()
```
Comment (tchaikov) [inline ccmlib/node.py:874]:
```suggestion
        env = self.get_env()
        if extra_env is not None:
            env.update(extra_env)
        return env
```
        else:
            env = self.get_env()
```

**Review by tchaikov — APPROVED** [review]
left a couple nit. just for better readability.  we are adding/overriding existing `env`. so `extra_env` might be able to pass this message to programmer better.

**Review by tchaikov — APPROVED** [review]
lgtm

**Review by bhalevy — APPROVED** [review]

File: ccmlib/node.py (hunk 2 above)
Comment (fruch) [reply ccmlib/node.py]: done

**Pull request merged**